basis/furnace/actions/actions.factor

```diff
@@ -51,7 +51,7 @@ CONSTANT: revalidate-url-key "__u"
 
 : revalidate-url ( -- url/f )
     revalidate-url-key param
-    dup [ >url ensure-port [ same-host? ] keep and ] when ;
+    [ >url ensure-port [ same-host? ] verify ] ?call ;
```

Comment:
so this is a WIP and verify has been renamed? Sorry was juste browsing some PRs without any specific goal